k2/csrc/dense_fsa.h

```diff
@@ -53,30 +52,31 @@ struct DenseFsa {
    */
   int32_t num_states() { return T + 2; }
```

Comment:
The method names of this `struct` do not follow Google C++ style.

Some methods should have a `const` modifier, such as `int32_t NumStates() const`.

Reply:
Sure